teams/nixcon.tt

```diff
@@ -7,6 +7,10 @@
   from any other OS/FOSS conference.
 </div>
 
+<div class="alert alert-info" style="display:inline-block; padding: 8px 14px;">
```

Comment:
```suggestion
<div class="alert alert-info">
```

I would leave this as default style of bootstrap as possible. You might want to add the title to the alert box, eg: "Help needed!"

Reply:
the default looks broken

![Screenshot from 2020-04-20 22-20-21](https://user-images.githubusercontent.com/91113/79796106-41588300-8355-11ea-8eb3-b3740830c529.png)

When i add "Help needed!", there is 2 times "help". any suggestion to better phrase it?

Reply:
To be honest this change doesn't really fit into this PR. Leave it out and maybe create a new PR and ask NixCon team what they actually want there.

Reply:
yeah, right